### PR TITLE
update rules

### LINKED ISSUE #190: Enable linter rules that can be automatically fixed
## Overview

The goal of this issue is to run all the rules that can be **automatically** fixed by running `lint --fix`.
The rules should follow https://github.com/telosnetwork/teloscan/issues/127

## Acceptance criteria

- Identify the rules that are automatically fixed from https://github.com/telosnetwork/teloscan/issues/127
- Run the fix
- Open the PR
- Create issues for the ones that can't be automatically fixed from https://github.com/telosnetwork/teloscan/issues/127

## Tasks

- [ ] [Default linting rules](https://github.com/telosnetwork/teloscan/issues/127) are applied
- [ ] #216 is merged
- [ ] All issues have been created

### PR DESCRIPTION
**Pull request by DevArtemNuss**
# Fixes  #190 
https://github.com/telosnetwork/teloscan/issues/127

## Description
Added new rulers for eslint 

   indent: ["error", 4],
    'max-len': ["warn", { "code": 120, "tabWidth": 2 }],
    'no-trailing-spaces' : ["error", { "skipBlankLines": true }],
    eqeqeq: ["error", "always"],   == -> ===
    semi: ["error", "always"],
    'eol-last': ["error", "never"],
    "vue/component-tags-order": ["error", {
      "order": [ [ "script", "template" ], "style" ] 
    }], 

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages

### CODE REVIEW

File: src/pages/accounts/add/SendOtp.vue
Comment (donnyquixotic) [inline src/pages/accounts/add/SendOtp.vue:45]: remove this commented code if no longer applicable

File: src/pages/accounts/add/VerifyOtp.vue
Comment (donnyquixotic) [inline src/pages/accounts/add/VerifyOtp.vue:32]: remove commented code

File: src/pages/accounts/add/VerifyOtp.vue
Comment (donnyquixotic) [inline src/pages/accounts/add/VerifyOtp.vue:36]: remove commented code

File: src/pages/profiles/add/EditImage.vue
Comment (donnyquixotic) [inline src/pages/profiles/add/EditImage.vue:99]: remove commented code

File: src/pages/profiles/list/ContactList.vue
Comment (donnyquixotic) [inline src/pages/profiles/list/ContactList.vue:76]: remove commented code

File: src/pages/profiles/list/ContactList.vue
Comment (donnyquixotic) [inline src/pages/profiles/list/ContactList.vue:70]: remove commented code

File: src/pages/profiles/read/ProfileDetail.vue
Comment (donnyquixotic) [inline src/pages/profiles/read/ProfileDetail.vue:89]: remove commented code

File: src/pages/profiles/read/ProfileDetail.vue
Comment (donnyquixotic) [inline src/pages/profiles/read/ProfileDetail.vue:108]: remove commented code

File: src/pages/profiles/read/ProfileDetail.vue
Comment (donnyquixotic) [inline src/pages/profiles/read/ProfileDetail.vue:118]: remove commented code

File: src/pages/trails/ballots/view/BallotView.vue
Comment (donnyquixotic) [inline src/pages/trails/ballots/view/BallotView.vue:49]: remove unused lifecycle hook

File: src/pages/works/components/ExpansionProposal.vue
Comment (donnyquixotic) [inline src/pages/works/components/ExpansionProposal.vue:75]: remove unused var

File: src/router/routes.js
Comment (donnyquixotic) [inline src/router/routes.js:9]: remove commented code

File: src/router/routes.js
Comment (donnyquixotic) [inline src/router/routes.js:17]: remove commented code

File: src/store/module-example/actions.js
Comment (donnyquixotic) [inline src/store/module-example/actions.js:1]: remove `module-example` directory

File: src/store/module-example/getters.js
Comment (donnyquixotic) [inline src/store/module-example/getters.js:1]: remove `module-example` directory

File: src/store/module-example/index.js
Comment (donnyquixotic) [inline src/store/module-example/index.js:12]: remove `module-example` directory

File: src/store/module-example/mutations.js
Comment (donnyquixotic) [inline src/store/module-example/mutations.js:1]: remove `module-example` directory

File: src/store/module-example/state.js
Comment (donnyquixotic) [inline src/store/module-example/state.js:5]: remove `module-example` directory

File: src/store/notifications/actions.js
Comment (donnyquixotic) [inline src/store/notifications/actions.js:4]: remove unused file

File: src/store/profiles/actions.js
Comment (donnyquixotic) [inline src/store/profiles/actions.js:1]: remove commented import

File: src/store/profiles/actions.js
Comment (donnyquixotic) [inline src/store/profiles/actions.js:102]: remove commented code

File: src/store/profiles/actions.js
Comment (donnyquixotic) [inline src/store/profiles/actions.js:149]: remove commented code

File: src/store/profiles/getters.js
Comment (donnyquixotic) [inline src/store/profiles/getters.js:14]: remove commented code

File: src/store/testnet/getters.js
Comment (donnyquixotic) [inline src/store/testnet/getters.js:4]: remove unused file

File: src/store/testnet/mutations.js
Comment (donnyquixotic) [inline src/store/testnet/mutations.js:4]: remove unused file

File: src/store/testnet/state.js
Comment (donnyquixotic) [inline src/store/testnet/state.js:3]: remove unused export/file

File: src/store/transfers/getters.js
Comment (donnyquixotic) [inline src/store/transfers/getters.js:4]: remove unused file

File: src/store/transfers/mutations.js
Comment (donnyquixotic) [inline src/store/transfers/mutations.js:4]: remove unused file

File: src/store/transfers/state.js
Comment (donnyquixotic) [inline src/store/transfers/state.js:3]: remove unused file

File: src/store/validators/actions.js
Comment (donnyquixotic) [inline src/store/validators/actions.js:25]: remove commented code

File: src/store/validators/actions.js
Comment (donnyquixotic) [inline src/store/validators/actions.js:48]: remove unused vars

File: src/store/validators/getters.js
Comment (donnyquixotic) [inline src/store/validators/getters.js:4]: remove unused file

**Review by donnyquixotic — CHANGES_REQUESTED** [review]

**Review by donnyquixotic — APPROVED** [review]

**Review by donnyquixotic — CHANGES_REQUESTED** [review]
still pending changes requested

**Review by donnyquixotic — APPROVED** [review]

File: .eslintrc.js
Comment (EJSG) [inline .eslintrc.js:84]: single/double should be consistent for the rules, eg. `'eol-last'` vs `"vue/component-tags-order"` (which I guess will take care of itself when a quotes rules is added haha)

**Review by EJSG — APPROVED** [review]
approved, looks good. optionally you can address my one comment about quote style consistency but that doesnt need to block merging

File: .eslintrc.js
Comment (Viterbo) [reply .eslintrc.js:84]: Somehow it didn't !!
So I fixed it personally for you (5b303e0c8e18f08103083a1578d4b9768750504e)

**Review by Viterbo — APPROVED** [review]

**Comment by Kersten-TCD** [comment]
@Viterbo I found an issue when clicking on Elections in top menu, this will not refresh the page and proposals are still shown. Same happens when selecting Proposals after refresh on elections page. I need to refresh page to see proposals again.

Will check if this can be reproduced on PR #227 once this PR is merged.

**Pull request merged**
